lib/matas_service_generator/service/templates/service.erb

```diff
@@ -1,18 +1,18 @@
-<% if @module_name.present? %> module <%= @module_name.camelize %>  <% end %>
-class <%= class_name %>
+<%- indent = @module_name.present? ? '  ' : '' -%>
```

Comment:
The changes you made look good for a service in a module with methods, but If I make a simple service, the indentation of the 'call' and 'initialize' does not seem right. What do you think of adding a second level indentation as stated down in this comment, and using that second_level_indentation everywhere except for 'class_name' and it's ending? If you have better idea, please do share. I wasn't aware of this in erb, but now I would like to make it look nice for each scenario :D 

```suggestion
<%- indent = @module_name.present? ? '  ' : '' -%>
<%- second_level_indent = indent + '  '  -%>
```

Reply:
Hmmm. I have tried it in all the documented combinations: with and without a module, with and without additional methods, and the only thing I see that looks wrong is the line-spacing around `private`. I need to add a newline after private to go with the one before. All the indents are two spaces, as per the Rubocop standard, and when you're inside a module, it's four spaces.